.github/workflows/github-pages.yml

```diff
@@ -50,9 +50,8 @@ jobs:
       env: 
         GPU_OCC_CALC: src/docs/_build/Tools/GPU-Occupancy-Calculator/
       run: |
-        mkdir -p ${GPU_OCC_CALC}   
-        cd ${GPU_OCC_CALC}
-        wget https://raw.githubusercontent.com/oneapi-src/oneAPI-samples/master/Tools/GPU-Occupancy-Calculator/index.html
+        mkdir -p ${GPU_OCC_CALC}
+        cp -v ${{ github.workspace }}/master/Tools/GPU-Occupancy-Calculator/index.html ${GPU_OCC_CALC}/index.html
 

```

Comment:
@pdxjohnny -- I'm fine with using this method by using github context variable (`github.workspace`).

First I need to know if you have tested it. ?

Reply:
Looks like we are good to go :)

```
'/home/runner/work/oneAPI-samples/oneAPI-samples/master/Tools/GPU-Occupancy-Calculator/index.html' -> 'src/docs/_build/Tools/GPU-Occupancy-Calculator//index.html'
```

Reply:
IM me asap, pls. 😺

Reply:
Incorporated 2 lines from this PR in https://github.com/oneapi-src/oneAPI-samples/pull/1585. 

You may now close this PR. Thanks for the contrib.